lib/deck.rb

```diff
@@ -0,0 +1,24 @@
+require './lib/card'
+
+class Deck
+  attr_reader :cards
+
+  def initialize(cards)
+    @cards = cards
+  end
+
+  def count
+    @cards.count
+  end
+
+  def cards_in_category(category)
+    cards_category = []
+    @cards.each do |card|
+      if card.category == category
+        cards_category << card
+      end
+    end
+    cards_category
+  end
```

Comment:
Today, we'll talk more about other enums you can use. You did a great job using each here, and this would be a good example of a time to use find_all.